src/languageServerPlugin.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+import * as commands from "./commands";
+
+export enum CompileWorkspaceStatus {
+    FAILED = 0,
```

Comment:
Do not using all upper case in enum: https://www.typescriptlang.org/docs/handbook/enums.html